### ISSUE TITLE
Request - add Boost

### ISSUE THREAD

**Issue by CagesThrottleUs** [brechtsanders/winlibs_mingw#144]
For C++ development, especially on WIndows, Boost Library is a god given. I request to please add this library and files in releases and provide a method so that these libraries can be searched by Compiler in PATH.

Webpage - https://www.boost.org/
Github - https://github.com/boostorg/boost

**Comment by CagesThrottleUs** [comment]
Hi @ldgeng sorry for the late reply, YES! Boost is extremely important for serious development and testing with C++. Rather than writing libraries again and again, a lot of stuff has been already implemented in boost. In fact a lot of modern C++ standard libraries have their roots in boost for example your `std::unordered_set` etc.

You can read more [here](https://stackoverflow.com/questions/358533/how-important-is-boost-to-learn-for-c-developers#358581) and [here](https://stackoverflow.com/questions/125580/what-are-the-advantages-of-using-the-c-boost-libraries#125922).

**Comment by michael-o** [comment]
> Hi @ldgeng sorry for the late reply, YES! Boost is extremely important for serious development and testing with C++. Rather than writing libraries again and again, a lot of stuff has been already implemented in boost. In fact a lot of modern C++ standard libraries have their roots in boost for example your `std::unordered_set` etc.
> 
> You can read more [here](https://stackoverflow.com/questions/358533/how-important-is-boost-to-learn-for-c-developers#358581) and [here](https://stackoverflow.com/questions/125580/what-are-the-advantages-of-using-the-c-boost-libraries#125922).
> 
> Also hello @michael-o, would appreciate your comments on why Boost should not be included rather than just an emoji reaction.

Plain simple: This is a compiler set, not a library set. Boost is a beast, even if a good one. If you need it, you can build it yourself since you get a decent compiler for it.

**Comment by brechtsanders** [comment]
Agreed, this is a library, not part of the compiler tools.

Having said that, I am able to build boost (see recipe [here](https://github.com/brechtsanders/winlibs_recipes/blob/main/recipes/boost.winlib)), and once the winlibs package manager is ready for general use it will be made available this way.
